BUILD_INSTRUCTIONS_FOR_LINUX.md

```diff
@@ -0,0 +1,39 @@
+# Build Instructions for Linux
+
+## Required Libraries
+*   Python2
+*   TensorFlow
+*   Git
+*   Bazel-3.7.2
+
+
+## Installation Instructions
+
+#### For Python2
+In the console, run:
+```bash
+sudo apt install python
+```
+
+#### For TensorFlow
```

Comment:
Let's just skip the python thing here, as developers usually have their own way to manage python (pyenv, virtualenv, conda, ...)